src/services/MoviesApi.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import axios from "axios";
+
```

Comment:
I guess you can combine these functions into one. 

`function getAll({sortOrder, querySearch, genre}) {`